zhtr.js

```diff
@@ -684,11 +684,11 @@ matchmaker.NoServersMatchQuery 遊戲更新中
 matchmaker.InvalidGameId 此伺服器已關閉或不存在。
 matchmaker.NoAvailableServers 目前沒有任何空閒的伺服器
 matchmaker.GameIssue 加入時出現未知問題
-adblocker 停用廣告攔截器以獲得免費KR輪盤。
+adblocker 禁用廣告攔截插件以獲得免費KR輪盤。
 ad-failed 廣告載入失敗，請稍後重試。
 streamers.views {0}個觀眾
 streamers.none 目前沒有任何Twitch直播！
-error.extentions 如果你有任何瀏覽器擴充功能，請停用它們
+error.extentions 如果你有任何瀏覽器插件，請停用它們
```

Comment:
https://zh.wikipedia.org/zh-tw/%E7%80%8F%E8%A6%BD%E5%99%A8%E6%93%B4%E5%85%85%E5%8A%9F%E8%83%BD

Reply:
Originally the line is just "extensions" without "browser", so 瀏覽器 should be omitted here. And for "extensions", the official term defined by NAER for it is 外掛程式, but nowadays the word 外掛 has been commonly known as "hacks" in the Chinese gaming community which, although they are related to each other in a way, is not exactly the same thing. Since one of the main focus of Chinese translations works is not to use any terms with ambiguity, I will go for 擴充程式 here, changing "功能" to "程式" since 功能 sounds weird here.

Reply:
* 擴充功能 (not 擴充程式)as Google Chrome refers it as that way.